client/css/style.css

```diff
@@ -1044,7 +1044,7 @@ kbd {
 	word-wrap: break-word;
 	word-break: break-word; /* Webkit-specific */
 	display: flex;
-	overflow: hidden;
```

Comment:
Need to keep this rule, at least on `.text`. Otherwise stuff like zalgo text is going to overflow which we don't want.

Reply:
Yeah, I mentioned that in my previous comment. I moved it to `.content` (it had no effect on `.text`, I'm guessing because it's an inline `span` :man_shrugging:).

Zalgo is now prevented again. Until we want to add a tooltip in a `.content`, for example a status message or something, and we'll be screwed again :D
But until then, that should fix it.